README.md

```diff
@@ -62,3 +54,118 @@ docker push forestadmin/lumber
 👇 Join our Slack community of +1000 developers
 
 [![Slack Status](http://community.forestadmin.com/badge.svg)](https://community.forestadmin.com)
+
+## Who Uses Forest Admin
```

Comment:
To move before "Contribute" section IMO (more important than "Contribute" and "Community")